scripts/benchmarks_ci.py

```diff
@@ -221,25 +222,37 @@ def __main(args: list) -> int:
 
     # Run micro-benchmarks
     if not args.build_only:
-        for framework in args.frameworks:
-            micro_benchmarks.run(
-                BENCHMARKS_CSPROJ,
-                args.configuration,
-                framework,
-                verbose,
-                args
-            )
-            
-        dotnet.shutdown_server(verbose)
-
-        if args.upload_to_perflab_container:
-            import upload
+        upload_container = UPLOAD_CONTAINER
+        try:
+            for framework in args.frameworks:
+                micro_benchmarks.run(
+                    BENCHMARKS_CSPROJ,
+                    args.configuration,
+                    framework,
+                    verbose,
+                    args
+                )
             globpath = os.path.join(
                 get_artifacts_directory() if not args.bdn_artifacts else args.bdn_artifacts,
                 '**',
                 '*perf-lab-report.json')
+        except CalledProcessError:
+            upload_container = 'failedresults'
+            globpath = os.path.join(
+                get_artifacts_directory(), 
+                'FailureReporter', 
+                'failure-report.json')
+            cmdline = [
+                'FailureReporting.exe', globpath
+            ]
+            RunCommand(cmdline, verbose=verbose).run(
```

Comment:
This path is not correct, you need to look back to the yml changes you made. We are saving the build executable to `$(CorrelationStaging)FailureReporter` which means that when we want to access it on the build machines we are going to need to look in the correlation payload directory. This is why Bill had mentioned the need to port over the helixpayload function from util.py to common.py. You will want to use that to get the proper directory that the FailureReporting.exe will be in. Also since you specify an out folder it will not use the `bin\FailureReporting\Debug\netcoreapp3.11, but will instead just be sitting in the folder you specified.

Reply:
Already fixed locally